### PR TITLE
SEMS output for measures and either-neither

### PR DESCRIPTION
**Pull request by benadida**

### CODE REVIEW

File: converter/SEMSoutput.py [hunk 1]
```diff
@@ -140,29 +174,69 @@ def add_entry(precinct_id, contest_id, answer):
...
 
         for contest in contests:
+            if contest['type'] == 'ms-either-neither':
+                either_neither_contest_id = contest["eitherNeitherContestId"]
+                pick_one_contest_id = contest["pickOneContestId"]
+
+                either_neither_answer = cvr_obj.get(either_neither_contest_id, None)
+                pick_one_answer = cvr_obj.get(pick_one_contest_id, None)
+
+                # both of these should be present, otherwise it's an invalid CVR and we shouldn't count this contest at all
```
Comment (eventualbuddha) [inline converter/SEMSoutput.py:184]: Is this true? I thought you could pick "neither" and leave the pick-one options blank.

Comment (eventualbuddha) [reply converter/SEMSoutput.py:184]: Ah, I guess "blank" would be an empty array?

Comment (benadida) [reply converter/SEMSoutput.py:184]: present as in it should be at least `[]`. `[]` means no user selection.

Comment (benadida) [reply converter/SEMSoutput.py:184]: I have a ticket assigned to you I think about making sure the scanner always outputs `contest_id: []` for blank contests that are on the page, which I think you're already doing.

File: converter/SEMSoutput.py [hunk 2]
```diff
@@ -140,29 +174,69 @@ def add_entry(precinct_id, contest_id, answer):
...
+                if either_neither_answer == []:
+                    add_entry(precinct_id, either_neither_contest_id, BLANKVOTE_CANDIDATE["id"])
+                else:
+                    if len(either_neither_answer) > 1:
+                        add_entry(precinct_id, either_neither_contest_id, OVERVOTE_CANDIDATE["id"])
+                    else:
+                        if either_neither_answer == ["no"] or pick_one_answer_valid:
+                            add_entry(precinct_id,
+                                      either_neither_contest_id,
+                                      contest["eitherOption"]["id"] if either_neither_answer == ["yes"] else contest["neitherOption"]["id"])
```
Comment (eventualbuddha) [inline converter/SEMSoutput.py:213]: nit: is it better to have these rather nested `if` statements?

File: converter/SEMSoutput.py (hunk 1 above)
Comment (eventualbuddha) [reply converter/SEMSoutput.py:184]: It should already be doing that, yeah.

File: converter/SEMSoutput.py (hunk 2 above)
Comment (benadida) [reply converter/SEMSoutput.py]: good call, cleaning up.

**Pull request merged**